### DISCUSSION TITLE
If we use open metadata how we can say our data is secure?

### DISCUSSION THREAD

**Discussion by LingamAnitha** [open-metadata/OpenMetadata#7980] [Q&A] [answered]
If we use open metadata how we can say our data is secure?

**Comment by harshach** [comment, marked as answer]
@LingamAnitha OpenMetadata is an open source project and all artifacts that you are downloads or docker images are controlled by your team.  For security scans find out the snyk report and github dependabot results. Apart from that OpenMetadata doesn't collect any data
All of the services run by your team in your infrastructure any of the data you stored in your OpenMetadata instance doesn't leave your infrastructure. Think of it like any other service you may run such as hive/kafka etc..

**Reply by tsalvesen2025** [reply]
Thank you, this is helpful; however, where can I find these reports for both snyk and dependabot for the latest version? My security team is asking me to provide penetration testing results and any related security certifications achieved by OpenMetadata, can this please be provided? Thank you.
